### ISSUE TITLE
Prettier key binding is overridden

### ISSUE THREAD

**Issue by sei40kr** [syl20bnr/spacemacs#11843]
I noticed Prettier key binding is overridden when used with lsp. Perhaps it needs another key binding such like `SPC m = p` and remove formatter option.

**Comment by thanhvg** [comment]
Yeah I just noticed this too. Both web-beautiful and prettier layers key binding `spc =` are overridden by lsp `spc = b` lsp-format-buffer. For js layer, 'lsp-format-buffer doesn't do anything at all. 

If we keep   `spc = b` for lsp I think we should rebind web-beautful and pretier to `spc = =`
